content/en/synthetics/browser_tests/_index.md

```diff
@@ -96,39 +102,39 @@ A notification is sent according to the set of alerting conditions. To configure
 
     Notification messages include the **message** defined in this section and information about the failing locations.
 
-2. Choose your [services][9] and/or team members to notify.
+2. Choose your [services][11] and/or team members to notify.
 3. Specify a renotification frequency. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
 4. Click **Save Details and Record Test**.
 
 ## Record your steps
 
-Tests can be only recorded from **[Google Chrome][10]**. To record your test, download the [Datadog Record Test extension for Google Chrome][11].
-
-{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test"  >}}
```

Comment:
Do we want to remove the image?
I would go for updating it
Displaying a img from a browser test recording with some steps inside

Reply:
That's fair; adding an updated screenshot in the following commit.